arrow/arrow.py

```diff
@@ -1057,6 +1057,83 @@ def humanize(
                 )
             )
 
+    def dehumanize(self, timestring):
+        """ Returns an arrow object relative to the humanized difference in time given.
+            Only works with specifically formatted strings for the time being.
+
+        :param timestring: A string in the format returned by humanize by default
+            Number must be before unit of time, i.e. (4 hours)
+
+        Usage:
+
+            >>> arrow.utcnow()
+            <Arrow [2020-04-17T00:57:03+00:00]>
+
+            >>> arrow.utcnow().dehumanize("4 hours ago")
+            <Arrow [2020-04-17T20:57:03+00:00]>
+
+            >>> arrow.utcnow().dehumanize("in 4 hours")
+            <Arrow [2020-04-18T04:57:03+00:00]>
+
+            >>> arrow.utcnow().dehumanize("4 days 7 hours 10 minutes 5 seconds ago")
+            <Arrow [2020-04-12T19:46:58+00:00]>
+        """
+
+        current = self.fromdatetime(self._datetime)
+
+        times = timestring.split(" ")
+
+        if times[-1] == "ago":
+            sign = -1
+            times = times[:-1]
+        elif times[0] == "in":
+            sign = 1
+            times = times[1:]
+        else:
+            raise ValueError("Invalid prefix or suffix")
+
+        if len(times) % 2 != 0:
+            raise ValueError("Invalid time input")
```

Comment:
Let's make these error messages more detailed, with a prompt on how the user can fix the problem.